src/gtil/predict.cc

```diff
@@ -37,6 +37,10 @@ namespace {
 using treelite::threading_utils::ThreadConfig;
 using PredTransformFuncType = std::size_t (*) (const treelite::Model&, const float*, float*);
 
+constexpr std::size_t kBlockOfRowsSize = 64;
+constexpr std::int32_t largest_representable_int
+  = (static_cast<std::int32_t>(1) << std::numeric_limits<float>::digits);
```

Comment:
Isn't this the same as the following?
```suggestion
auto static constexpr const largest_representable_int = std::numeric_limits<std::int32_t>::max();
```

Reply:
This variable is actually not used. Let me remove it.